docs/langref/src/concepts/layouting.md

```diff
@@ -100,19 +100,19 @@ following properties:
 
 A layout element also affects the minimum and maximum size of its parent.
 
-An element is considered to have a fixed size in a layout when the `width` and `height` is specified directly.
+Any element with a specified `width` and `height` has a fixed size in a layout.
```

Comment:
Nice simplification, and also a consistent use of "fixed" instead of say "has a hard-coded size in a layout" :-)